src/traces/sankey/defaults.js

```diff
@@ -24,6 +25,11 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     coerce('node.thickness');
     coerce('node.line.color');
     coerce('node.line.width');
+    coerce('node.hoverinfo');
+    Registry.getComponentMethod(
```

Comment:
I could have used a one-liner and call `var Fx = require('../../components/fx'); Fx.supplyDefaults (...)`. I used the Registry but don't know which one is better 🤷‍♂️.

Reply:
`Registry` is useful when a component may or may not be in the bundle, or to break circular references. I can't really see us making `Fx` optional any time soon, so I'd probably just require the defaults only:
`var handleFxDefaults = require('../../components/fx/defaults');`

That said, it would be even better to dive even one level deeper and call `handleHoverLabelDefaults` directly, using new `coerce` functions based on the one used here, like how `polar` makes `coerceAxis`:

https://github.com/plotly/plotly.js/blob/c922c61234977756d04598a29d448dd7efb74c63/src/plots/polar/layout_defaults.js#L42-L44

Two reasons for this: it'll work with templates (without having to propagate the template explicitly into `node` and `link`, which doesn't look like it's been done here, see eg [Etienne's recent carpet fix](https://github.com/plotly/plotly.js/pull/3095/files#diff-55e73e1ce7c046db1f44dbeb16ce40caR38)), and it's actually using the attributes defined in this trace in case we ever need them to diverge from the ones in `fxAttrs`

Reply:
I am not sure I'm properly addressing this comment. Can you check commit fe7ab49 and tell me whether this is what you had in mind? Thanks @alexcjohnson